walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -99,9 +104,11 @@ public void everyAttendeeIsConsidered() {
     // Have each person have different events. We should see two options because each person has
     // split the restricted times.
     //
-    // Events  :       |--A--|     |--B--|
-    // Day     : |-----------------------------|
-    // Options : |--1--|     |--2--|     |--3--|
+    // Mandatory: A, B
+    // Events   :       |--A--|
```

Comment:
Nit: Call this "Person A's Events" and the below one "Person B's Events" to make it a bit clearer.